modules/document_repository/ajax/documentDelete.php

```diff
@@ -67,4 +72,18 @@
     unlink($path);
 }
 
+// Cleanup empty directories
+set_error_handler(
+    function () {
+        // Silence the E_WARNING when files exist in the directory.
```

Comment:
I don't know if this is the best way to do this. Shouldn't we delete the files first in this case? I'm a little worried about this silencing useful warnings.